src/pages/_proxied-dot-io/waypoint/downloads/index.tsx

```diff
@@ -0,0 +1,70 @@
+import WaypointIoLayout from 'layouts/_proxied-dot-io/waypoint'
+import ProductDownloadsPage from '@hashicorp/react-product-downloads-page'
+import {
+  generateStaticProps,
+  ReleasesAPIResponse,
+} from 'lib/fetch-release-data'
+import styles from './style.module.css'
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+function WaypointDownloadsPage({
+  latestVersion,
+  releases,
+}: {
+  latestVersion: string
+  releases: ReleasesAPIResponse
+}) {
+  const getStartedDescription =
+    'Follow step-by-step tutorials on AWS, Azure, GCP, and localhost.'
+  const getStartedLinks = [
+    {
+      label: 'Deploy to Docker',
+      href:
+        'https://learn.hashicorp.com/collections/waypoint/get-started-docker',
+    },
+    {
+      label: 'Deploy to Kubernetes',
+      href:
+        'https://learn.hashicorp.com/collections/waypoint/get-started-kubernetes',
+    },
+    {
+      label: 'Deploy to AWS',
+      href: 'https://learn.hashicorp.com/collections/waypoint/deploy-aws',
+    },
+    {
+      label: 'View all Waypoint tutorials',
+      href: 'https://learn.hashicorp.com/waypoint',
+    },
+  ]
+  const logo = ( // eslint-disable-next-line @next/next/no-img-element
+    <img
+      className={styles.logo}
+      alt="Waypoint"
+      src={require('./img/waypoint-logo.svg')}
+    />
+  )
+  const tutorialLink = {
+    href: 'https://learn.hashicorp.com/waypoint',
+    label: 'View Tutorials at HashiCorp Learn',
+  }
+
+  return (
+    <ProductDownloadsPage
+      enterpriseMode={false}
```

Comment:
Ack, frustrating that `enterpriseMode` & `merchandisingSlot` are required in the underlying component, when I think they might not need to be 🤔  but that's def outside the scope of this PR. Very minor non-blocking nit, but I think we could add a comment here if we wanted to mention that (if it is indeed the case, I could be wrong here).

Reply:
I think `enterpriseMode` should default to `false` that `merchandisingSlot` should be optional. I think we're on the same page! I'll create an Asana task for updating `ProductDownloadsPage` (or at least to be triaged tomorrow) and I'll link to it here in a TODO comment.